packages/framework/src/Resources/views/Admin/Inline/Icon/_info_icon.html.twig

```diff
@@ -0,0 +1,6 @@
+{{ ux_icon('tabler:info-circle-filled', {
```

Comment:
why is there an underscore in the filename? `_info...` I do not mind that, it is just unusual in our codebase :shrug:

Reply:
it's a convention to distinguish between full templates and template fragments
https://symfony.com/doc/current/templates.html#including-templates

And now I'm torn whether the function template is a fragment or full template 😅 

... okay it seems like it should not be named with the leading underscore. Changed in f7f9a6e59f